src/admin/components/SearchForm.js

```diff
@@ -3,57 +3,68 @@ import { connect } from 'react-redux';
 
 import { fetchAddon } from 'core/api';
 import { loadEntities } from 'core/actions';
+import { SearchFormBase } from 'core/components/Search/SearchForm';
 import { gettext as _ } from 'core/utils';
 
-import 'admin/css/SearchForm.scss';
-import 'admin/css/lib/buttons.scss';
+import 'core/css/SearchForm.scss';
+import 'core/css/lib/buttons.scss';
 
-export class SearchFormBase extends React.Component {
+
+export function mapStateToProps({ api }) {
```

Comment:
Oh, okay, I think they were the other way around elsewhere and I liked seeing `render` at the bottom, but I'll move it back, no worries.